packages/cli/generation/local-generation/local-workspace-runner/src/runLocalGenerationForWorkspace.ts

```diff
@@ -128,26 +137,37 @@ async function writeIrToFile({
     generatorInvocation,
     workspaceTempDir,
     context,
+    irOverrideVersion,
 }: {
     workspace: FernWorkspace;
     audiences: Audiences;
     generatorInvocation: GeneratorInvocation;
     workspaceTempDir: DirectoryResult;
     context: TaskContext;
+    irOverrideVersion?: string;
```

Comment:
lets type this as `string | undefined` instead of `?: string` so that all callers have to specify the key

Reply:
ok, but won't this break the current code that calls the function?

Reply:
yup and you should update that code